tools/deploymentsSchemaTests/deploymentsSchemaTests.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Deployments.Templates" Version="1.0.154" />
```

Comment:
This reminds me - we should probably onboard this repo for dependabot, so that we pick up updates automatically. Here's how we have it configured for Bicep: https://github.com/Azure/bicep/blob/main/.github/dependabot.yml. I'll create an issue for this.